mixer/cmd/build.go

```diff
@@ -0,0 +1,162 @@
+// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/clearlinux/mixer-tools/builder"
+	"github.com/clearlinux/mixer-tools/helpers"
+	"github.com/pkg/errors"
+
+	"github.com/spf13/cobra"
+)
+
+type buildCmdFlags struct {
+	format     string
+	increment  bool
+	minVersion int
+	noSigning  bool
+	prefix     string
+	noPublish  bool
+	keepChroot bool
+	template   string
+}
+
+var buildFlags buildCmdFlags
+
+func buildChroots(builder *builder.Builder, signflag bool) error {
```

Comment:
I thought it odd this function lived in main.go; should it continue to live in build.go? Though I get that this is a 1:1 port, so I'm fine with that being a cleanup for later.

Reply:
Yes all the build XYZ functions should be placed in the build.go file.